### PR TITLE
adds invoice_items model and tests relationships

### LINKED ISSUE #4: Invoice_Items Model

### PR DESCRIPTION
**Pull request by Daniel-Gallagher92**

### CODE REVIEW

File: app/models/invoice_item.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class InvoiceItem < ApplicationRecord 
+  belongs_to :item
+  belongs_to :invoice
```
Comment (ethanrossblack) [inline app/models/invoice_item.rb:3]: Do we need to add the `enum` for status here?

**Review by JorjaF — APPROVED** [review]
looks good I approve

**Review by ethanrossblack — APPROVED** [review]

**Review by z-fitch — APPROVED** [review]
nice!

**Comment by JorjaF** [comment]
love it

**Pull request merged**